src/request/create/Create.js

```diff
@@ -21,114 +17,74 @@ class Create {
-        const $downloadKeyfile = (document.querySelector('.download-keyfile'));
-
-        /** @type {HTMLElement} */
-        const $privacyAgent = (document.querySelector('.privacy-agent'));
-
-        /** @type {HTMLElement} */
-        const $recoveryWords = (document.querySelector('.recovery-words'));
-        /** @type {HTMLElement} */
-        const $recoveryWordsButton = (document.querySelector('#recovery-words button'));
-        /** @type {HTMLElement} */
-        const $recoveryWordsSkip = (document.querySelector('#recovery-words .skip'));
+        /** @type {HTMLFormElement} */
+        this.$setPassphrasePage = (document.getElementById('set-passphrase'));
 
-        /** @type {HTMLElement} */
-        const $validateWords = (document.querySelector('.validate-words'));
+        /** @type {HTMLFormElement} */
+        this.$walletIdentifier = (document.querySelector('.wallet-identifier'));
 
         // Create components
 
-        this._identiconSelector = new IdenticonSelector($identiconSelector, request.defaultKeyPath);
-        this._downloadKeyfile = new DownloadKeyfile($downloadKeyfile);
-        this._privacyAgent = new PrivacyAgent($privacyAgent);
-        this._recoveryWords = new RecoveryWords($recoveryWords, false);
-        this._validateWords = new ValidateWords($validateWords);
+        this._identiconSelector = new IdenticonSelector(this.$identiconSelector, request.defaultKeyPath);
         this._passphraseSetter = new PassphraseSetterBox($setPassphrase);
+        // Set up progress indicators
+        /* eslint-disable-next-line no-new */
+        new ProgressIndicator(document.querySelector(`#${Create.Pages.CHOOSE_IDENTICON} .progress-indicator`), 3, 1);
+        this.progressIndicator = new ProgressIndicator(
+            document.querySelector(`#${Create.Pages.SET_PASSPHRASE} .progress-indicator`),
+            3,
+            2,
+        );
 
         // Wire up logic
 
         this._identiconSelector.on(
             IdenticonSelector.Events.IDENTICON_SELECTED,
-            /** @param {Nimiq.Entropy} entropy */
-            entropy => {
+            /**
+             * @param {Nimiq.Entropy} entropy
+             * @param {string} address
+            */
+            (entropy, address) => {
                 this._selectedEntropy = entropy;
-                const mnemonic = Nimiq.MnemonicUtils.entropyToMnemonic(entropy);
-                this._recoveryWords.setWords(mnemonic);
-                this._validateWords.setWords(mnemonic);
                 window.location.hash = Create.Pages.SET_PASSPHRASE;
+                // eslint-disable-next-line no-new
+                new Identicon(
+                    address,
+                    /** @type {HTMLDivElement} */(this.$walletIdentifier.querySelector('.identicon')),
+                );
+                this.progressIndicator.setStep(2);
                 this._passphraseSetter.reset();
                 this._passphraseSetter.focus();
```

Comment:
I think this `.focus()` also needs the logic about no autofocus on mobile.

Reply:
It does, as do lots of other `.focus()` calls. I don't think maintaining that on many branches helps. I will open a new PR once the constant is merged.